src/registrar/forms/application_wizard.py

```diff
@@ -389,10 +539,32 @@ def get_template_names(self):
         """
         return [TEMPLATES[self.steps.current]]
 
+    def _is_federal(self) -> Union[bool, None]:
+        """Return whether this application is from a federal agency.
+
+        Returns True if we know that this application is from a federal
+        agency, False if we know that it is not and None if there isn't an
+        answer yet for that question.
+        """
+        organization_type_data = self.get_cleaned_data_for_step("organization_type")
+        if organization_type_data is None:
+            return None  # no answers here yet
+        organization_type = organization_type_data.get("organization_type")
+        if organization_type is None:
+            # they haven't answered this question
+            return None
+        else:
+            # they have answered this question
+            if organization_type == DomainApplication.OrganizationChoices.FEDERAL:
+                return True
+            return False
+
     def get_context_data(self, form, **kwargs):
         """Add title information to the context for all steps."""
         context = super().get_context_data(form=form, **kwargs)
         context["form_titles"] = TITLES
+        if self.steps.current == "organization_contact":
```

Comment:
```suggestion
        if self.steps.current == Step.ORGANIZATION_CONTACT:
```

Came back to reference this PR for the one I'm working on and saw a **nit** I missed on my first pass.